fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/MatcherHelper.kt

```diff
@@ -111,8 +111,10 @@ object MatcherHelper {
         for (appItem in appItems) {
             if (exists) {
                 assertTrue(appItem.waitForExists(waitingTime))
+                Log.i(TAG, "assertItemWithResIdExists: Verified ${appItem.selector} exists")
             } else {
                 assertFalse(appItem.waitForExists(waitingTimeShort))
```

Comment:
Same as above: "Failure message in case it does exist"

Reply:
Done !